tajo-core/src/main/java/org/apache/tajo/engine/planner/physical/HashJoinExec.java

```diff
@@ -27,9 +28,12 @@
 
 public class HashJoinExec extends CommonHashJoinExec<TupleList> {
 
+  private final boolean isCrossJoin;
```

Comment:
The variable for checking cross join of ``CommonHashJoinExec`` have been deleted in this PR. How about add it to ``CommonHashJoinExec`` again?